src/topology/ping_pong.rs

```diff
@@ -0,0 +1,862 @@
+// SPDX-License-Identifier: MPL-2.0
+
+//! Implements the Ping-Pong Topology described in [VDAF]. This topology assumes there are exactly
+//! two aggregators, designated "Leader" and "Helper". Note that while this implementation is
+//! compatible with VDAF-06, it actually implements the ping-pong wrappers specified in VDAF-07
+//! (forthcoming) since those line up better with the VDAF implementation in this crate.
```

Comment:
These references will be fixed up once VDAF-07 is published as part of https://github.com/divviup/libprio-rs/issues/701